api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -674,14 +676,42 @@ def pick_up_tip(
         :param presses: The number of times to lower and then raise the pipette
                         when picking up a tip, to ensure a good seal (0 [zero]
                         will result in the pipette hovering over the tip but
-                        not picking it up--generally not desireable, but could
+                        not picking it up--generally not desirable, but could
                         be used for dry-run).
         :type presses: int
         :param increment: The additional distance to travel on each successive
                           press (e.g.: if `presses=3` and `increment=1.0`, then
                           the first press will travel down into the tip by
                           3.5mm, the second by 4.5mm, and the third by 5.5mm).
         :type increment: float
+        :param prep_after: Whether the pipette plunger should prepare itself
+                           to aspirate immediately after picking up a tip.
+
+                           .. warning::
+                               This is provided for compatibility with older
+                               Python Protocol API behavior. You should normally
+                               leave this unset.
+
+                           If ``True``, the pipette will move its plunger position to
+                           bottom in preparation for any following calls to
+                           :py:meth:.aspirate.
+
+                           If ``False``, the pipette will prepare its plunger later,
+                           during the next call to :py:meth:.aspirate. This is
```

Comment:
```suggestion
                           If ``True``, the pipette will move its plunger position to
                           bottom in preparation for any following calls to
                           :py:meth:`.aspirate`.

                           If ``False``, the pipette will prepare its plunger later,
                           during the next call to :py:meth:`.aspirate`. This is
```

Touching up the RST syntax here.